src/applications/static-pages/cta-widget/components/CallToActionAlert.jsx

```diff
@@ -14,7 +14,7 @@ export default function CallToActionAlert({
 }) {
   const CustomHeaderLevel = `h${headerLevel}`;
   return (
-    <va-alert visible status={status}>
+    <va-alert visible status={status} usdws>
```

Comment:
I think you meant for this to be `uswds`?

Reply:
Thank you, updated!